src/ucp/core/ucp_worker.inl

```diff
@@ -261,4 +261,26 @@ ucp_worker_default_address_pack_flags(ucp_worker_h worker)
         } \
     }
 
+/**
+ * Increase number of requests to create endpoint.
+ *
+ * @param [in]  worker  UCP worker.
+ */
+static UCS_F_ALWAYS_INLINE void
+ucp_worker_increase_num_ep_creations(ucp_worker_h worker)
+{
+    ++worker->num_ep_creations;
+}
+
+/**
+ * Increase number of failed requests to create endpoint.
+ *
+ * @param [in]  worker  UCP worker.
+ */
+static UCS_F_ALWAYS_INLINE void
+ucp_worker_increase_num_ep_creation_failures(ucp_worker_h worker)
```

Comment:
these functions used once - not sure need inline wrapper (we maybe have more of these)